services/core/src/sourcify-chains.ts

```diff
@@ -148,5 +148,22 @@ export default {
         "supported": true,
         "monitored": true,
         "graphQLFetchAddress": "https://rinkeby-indexer.arbitrum.io/graphql"
+    "40": {
+        "fullnode": {
```

Comment:
You probably don't need the `fullnode` property. The RPC is already specified in [chains.json](https://github.com/ethereum/sourcify/blob/master/services/core/src/chains.json), which was last updated a few days ago and includes the required data.

Reply:
Removed from both networks.